src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -9,7 +9,7 @@ import {convertString} from '../../../utils/string/StringUtils';
 import {colors} from '../../../utils/colors';
 
 const TopicDomainHeader = (props) => {
-  const {handleOnMemberPress, hideSeeMember, isFollow, memberCount, topicDetail} = props;
+  const {domain, handleOnMemberPress, hideSeeMember, isFollow, memberCount} = props;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `topicDetail` prop has been replaced with a `domain` prop. Ensure that all parent components passing this prop have been updated accordingly to avoid any undefined errors.
```diff
-  const {handleOnMemberPress, hideSeeMember, isFollow, memberCount, topicDetail} = props;
+  const {domain, handleOnMemberPress, hideSeeMember, isFollow, memberCount} = props;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->